profiles/factories.py

```diff
@@ -73,7 +73,7 @@ def state_or_territory(self):
         if self.country not in {"US", "CA"}:
             return ""
         subdivisions = pycountry.subdivisions.get(country_code=self.country)
-        subdivision = random.randgen.sample(subdivisions, 1)[0]
+        subdivision = random.randgen.sample(list(subdivisions), 1)[0]
```

Comment:
What was the reason for the casting explicitly?

Reply:
I was getting this depreciation warning:
` DeprecationWarning: Sampling from a set deprecated
  since Python 3.9 and will be removed in a subsequent version.
    subdivision = random.randgen.sample(subdivisions, 1)[0]`